### PR TITLE
Vrp porting

### PR DESCRIPTION
**Pull request by taiuti**
### Referenced pull requests

I ported the classic example [vehicle routing](https://github.com/kiegroup/optaplanner/tree/main/optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting) to quarkus taking it as an example [facility-location](https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/facility-location).

I hope it is of interest to the project

Marco

### CODE REVIEW

**Comment by triceo** [comment]
Thank you for the contribution, @taiuti!
At the moment, we are not sure if we want to move all classic examples into quickstarts, therefore I can not promise if/when we will merge your PR.

We will discuss this topic when more team members are back from their summer vacations, and we will get back to you after we have.

File: use-cases/vehicle-routing/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  </div>
+</div>
+
+<div class="modal fade" id="scoreDialog" tabindex="-1" role="dialog">
+  <div class="modal-dialog modal-lg" role="dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5>Score explanation</h5>
+        <button type="button" class="close" data-dismiss="modal">
+          <span>&times;</span>
```
Comment (dupliaka) [inline use-cases/vehicle-routing/src/main/resources/META-INF/resources/index.html:105]: Shell it stay empty?

Comment (yurloc) [reply use-cases/vehicle-routing/src/main/resources/META-INF/resources/index.html:105]: No, it shouldn't, but it is a minor issue that I'm planning to fix myself after the PR is merged.

**Review by yurloc — COMMENTED** [review]
@taiuti Looks good! It needs some cleanup but I'll take care of that after it's merged.

Sorry for the inconvenience but we made some structural changes to the repository before we got to make a decision about this PR. Do you think you could take care of resolving the conflicts? If not, I can do that. I'll just need to open a new PR but I'll preserve your commits so you'll get the credit for the contribution.

**Comment by taiuti** [comment]
@yurloc I will try to make changes on this weekend, if I had problems I will ask you for help

**Review by yurloc — APPROVED** [review]
Looks good!

The conflicts I mentioned still exist but they only affect the "rebase" strategy. I'm going to squash the PR to avoid the conflicts.

File: use-cases/vehicle-routing/src/main/resources/META-INF/resources/app.js [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    //L.polyline([customer.location, customer.vehicle.location], {color}).addTo(customerGroup);
+  });
+
+  //Route
+  solution.vehicleList.forEach((vehicle) => {
+    const color = colorByVehicle(vehicle);
+    from = vehicle.depot.location;
+    isRouteValid = false;
+    vehicle.route.forEach((route) => {
+      isRouteValid = TextTrackCueList;
```
Comment (yurloc) [inline use-cases/vehicle-routing/src/main/resources/META-INF/resources/app.js:248]: @taiuti What did you mean by this? It's clearly a typo but I don't know how to fix it :slightly_frowning_face: Can you give me a hint how you intended to decide whether a route is valid or not?

Comment (yurloc) [reply use-cases/vehicle-routing/src/main/resources/META-INF/resources/app.js:248]: Never mind, I've found out I can draw the whole route with a single `L.polyLine()` call.

Comment (taiuti) [reply use-cases/vehicle-routing/src/main/resources/META-INF/resources/app.js:248]: > @taiuti What did you mean by this? It's clearly a typo but I don't know how to fix it slightly_frowning_face Can you give me a hint how you intended to decide whether a route is valid or not?

@yurloc yes, it's  a typo .
The idea is use isRouteValid variable for draw the last line to depot only if  the exist at least one customer to visit .
The right code is 

      ` isRouteValid = TextTrackCueList; ` =>  `    isRouteValid = true;`

**Pull request merged**